app/styles/ember-emeis.scss

```diff
@@ -18,3 +18,15 @@
   cursor: pointer;
   outline: inherit;
 }
+
+.hidden-input {
+  height: 0;
+  width: 100%;
+  border: none;
+  display: block;
+  font-size: 0;
+}
```

Comment:
Just use `<input type="hidden">`

Reply:
A input of type hidden is not validated on form submission. This is the workaround so we still have html validation for a power select field.